tests/Aspire.Hosting.Valkey.Tests/ValkeyPublicApiTests.cs

```diff
@@ -0,0 +1,123 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Xunit;
+
+namespace Aspire.Hosting.Valkey.Tests;
+
+public class ValkeyPublicApiTests
+{
+    #region ValkeyBuilderExtensions
+
+    [Fact]
+    public void AddValkeyContainerShouldThrowsWhenBuilderIsNull()
+    {
+        IDistributedApplicationBuilder builder = null!;
+        const string name = "Valkey";
+
+        var action = () => builder.AddValkey(name);
+
+        Assert.Multiple(() =>
```

Comment:
Based on this [comment](https://github.com/dotnet/aspire/pull/5099#discussion_r1696261977). Assert.Multyple is not necessary.

You should just leave it here:
var exception = Assert.Throws<ArgumentNullException>(action);
Assert.Equal(nameof(builder), exception.ParamName);

Same for the code below